unit12/index.js

```diff
@@ -0,0 +1,77 @@
+//ex1 
+function isEqual(a, b) {
+    if (a === b) return true;
+    if (a == null || b == null) return false;
+    if (a.length !== b.length) return false;
+      for (var i = 0; i < a.length; ++i) {
+      if (a[i] !== b[i]) return false;
+    }
```

Comment:
```suggestion
    if (a.length !== b.length) return false;

    for (let i = 0; i < a.length; ++i) {
      if (a[i] !== b[i]) return false;
    }
```

Lần sau còn dùng `var` thì ko review nữa